pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
+
+import { Btn } from "@/components/Button";
+
+import React, { useState } from "react";
+
+import myComponent from "@/components/myComponent1/myComponent.js";
+
+const kosmetikProdukt = [
+  {
+    name: "Luminous Silk Foundation",
+    brand: "Aurora",
+    type: "Foundation",
+    shade: "Porcelain",
+    benefits: ["hydrating", "long-lasting", "lightweight"],
+    price: "45€",
+    size: "30ml",
+  },
+  {
+    name: "Velvet Matte Lipstick",
+    brand: "Bloom Beauty",
+    type: "Lipstick",
+    shade: "Crimson Red",
+    benefits: ["moisturizing", "high pigment", "smudge-proof"],
+    price: "32€",
+    size: "15g",
+  },
+  {
+    name: "Ultra Defined Mascara",
+    brand: "Lash Luxe",
+    type: "Mascara",
+    shade: "Jet Black",
+    benefits: ["volumizing", "lengthening", "curling"],
+    price: "29€",
+    size: "10ml",
+  },
+  {
+    name: "Sunset Glow Blush",
+    brand: "Cheek Chic",
+    type: "Blush",
+    shade: "Peach Shimmer",
+    benefits: ["buildable", "natural-looking", "contains vitamin E"],
+    price: "25€",
+    size: "20g",
+  },
+  {
+    name: "Golden Hour Highlighter",
+    brand: "Gleam Glow",
+    type: "Highlighter",
+    shade: "Golden Peach",
+    benefits: ["radiant finish", "long-wear", "blendable"],
+    price: "38€",
+    size: "18g",
+  },
+];
+
+const StyledDiv = styled.div`
```

Comment:
just a suggestion. find a meaningful name:
```suggestion
const StyledContainer = styled.div`
```